packages/card/src/vaadin-card.js

```diff
@@ -31,11 +57,233 @@ class Card extends ElementMixin(ThemableMixin(PolylitMixin(LitElement))) {
   static get styles() {
     return css`
       :host {
+        display: flex;
+        flex-direction: column;
+        padding: var(--_padding);
```

Comment:
Should we set `box-sizing: border-box` on the card in core styles? Currently trying to set e.g. `width: 300px` will end up with the actual width being set to `332px` due to the default padding.

Reply:
Added `box-sizing: border-box`.